include/swift/Sema/ConstraintSystem.h

```diff
@@ -6137,7 +6137,7 @@ class DisjunctionChoice {
 
   void print(llvm::raw_ostream &Out, SourceManager *SM, unsigned indent = 0) const {
     Out << "disjunction choice ";
-    Choice->print(Out, SM);
+    Choice->print(Out, SM, /*skipLocator*/ false, /*skipReference*/ true);
```

Comment:
I think it's still useful to print this until we get "Changes" after each type variable and binding set otherwise it won't be possible to determine what type requirements did overload introduce to the system...

Reply:
+1 on that =]